components/icons/index.js

```diff
@@ -37,4 +37,5 @@ export { default as NettestGroupWebsites } from './NettestGroupWebsites'
 export { default as NettestTelegram } from './NettestTelegram'
 export { default as NettestWhatsApp } from './NettestWhatsApp'
 export { default as NettestVanillaTor } from './NettestVanillaTor'
+export { default as NettestGroupCircumvention } from './NettestGroupCircumvention'
```

Comment:
Maybe this should be placed above together with the other `NettestGroup` components?